### PR TITLE
Contact us and About us links

### LINKED ISSUE #1793: As a user, I want the "About Us" and "Contact Us" links to be easily accessible for navigation.
The contact form should be accessible when a user is on the homepage.

## Proposal

On the homepage, just under the About Us banner:

* put a button "Contact Us" that links to the Contact Us part of the about page 
* move the social media buttons right; perhaps match how they will appear in the footer (see below)

Adjust the footer as follows:

- links for about us; contact us
- change the social media links to explicit words, instead of logos, and move them under About us
- fix padding so that columns have padding between them on mobile screens

### PR DESCRIPTION
**Pull request by actlikewill**
- adds contact us and about us links

image

image

image

image

image

closes #1793

### CODE REVIEW

File: peachjam/templates/peachjam/_social_media.html [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
-             alt="LinkedIn"
-             width="40"
-             height="40"/>
-      </a>
-    </li>
-  {% endif %}
-</ul>
+<div class="row align-items-center justify-content-between">
+  <div class="col-md-auto">
+    <a class="btn btn-primary" href="{% url 'about' %}#contact">Contact us</a>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_social_media.html:4]: Translateable

**Comment by longhotsummer** [comment]
what does it look like on mobile?

File: peachjam/templates/peachjam/_social_media.html (hunk 1 above)
Comment (actlikewill) [reply peachjam/templates/peachjam/_social_media.html]: added

**Comment by actlikewill** [comment]
@longhotsummer I've added some screenshots for mobile. I added some responsive classes

**Pull request merged**